vulkaninfo/vulkaninfo.h

```diff
@@ -1278,10 +1278,9 @@ struct AppGpu {
         supported_format_ranges = {
             {
                 // Standard formats in Vulkan 1.0
-                VK_MAKE_VERSION(1, 0, 0),
-                NULL,
-                VK_FORMAT_BEGIN_RANGE,
-                VK_FORMAT_END_RANGE,
+                VK_MAKE_VERSION(1, 0, 0), NULL,
+                static_cast<VkFormat>(0),   // VK_FORMAT_UNDEFINED
+                static_cast<VkFormat>(184)  // VK_FORMAT_ASTC_12x12_SRGB_BLOCK
```

Comment:
Is it bad to just use UNDEFINED and ASTC_12x12_SRGB_BLOCK instead?

Reply:
I wanted to make it clear that its using the full range of formats as defined in the header, but it may be clearer to explicitly add that in a comment.